README.md

```diff
@@ -1317,6 +1317,7 @@ listed below.
 - Under section **(1) Check the IP Whitelist**, click on **ALLOW ACCESS FROM ANYWHERE**. The form will add a field with `0.0.0.0/0`.  Click **SAVE** to save the `0.0.0.0/0` whitelist.
 - Under section **(2) Choose a connection method**, click on **Connect Your Application**
 - In the new screen, click on **Standard connection string (3.4+ driver)**. __*WARNING*__: Do not pick 3.6+ since there Express Session currently has a compatibility issue with it.
+- In the new screen, select **Node.js** as Driver and version **2.2.12 or later**. _*WARNING*_: Do not pick 3.0 or later since connect-mongo can't handle mongodb+srv:// connection strings. Your connection string should look like `mongodb://<username>:<password>@cluster0-shard-00-00-ab2cd.mongodb.net:27017,cluster0-shard-00-01-ab2cd.mongodb.net:27017,cluster0-shard-00-02-ab2cd.mongodb.net:27017/test?ssl=true&replicaSet=Cluster0-shard-0&authSource=admin&retryWrites=true` (Ensure it starts as mongodb and not mongodb+srv)
```

Comment:
This is getting too long and might be hard to read.  Let's keep the first part and remove the part starting "Your connection string should look like ..."

Reply:
Agreed and updated.